src/platform/forms-system/src/js/components/Telephone.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// import recordEvent from 'platform/monitoring/record-event.js';
+
+export const CONTACTS = Object.freeze({
+  '222_VETS': '8772228387', // VA Help Line
+  '4AID_VET': '8774243838', // National Call Center for Homeless Veterans
+  911: '911',
+  CAREGIVER: '8552603274', // VA National Caregiver Support Line
+  CRISIS_LINE: '8002738255', // Veterans Crisis hotline
+  CRISIS_TTY: '8007994889', // Veterans Crisis hotline TTY
+  DS_LOGON: '8005389552', // Defense Manpower Data Center
+  DS_LOGON_TTY: '8663632883', // Defense Manpower Data Center TTY
+  GI_BILL: '8884424551', // Education Call Center (1-888-GI-BILL-1)
+  GO_DIRECT: '8003331795', // Go Direct/Direct Express (Treasury)
+  HELP_DESK: '8555747286', // VA Help desk
+  HELP_TTY: '8008778339', // VA Help Desk TTY
+  MY_HEALTHEVET: '8773270022', // My HealtheVet help desk
+  NCA: '8005351117', // National Cemetery Scheduling Office
+  TESC: '8882242950', // U.S. Treasury Electronic Payment Solution Center
+  VA_311: '8446982311', // VA Help desk (VA311)
+  VA_BENEFITS: '8008271000', // Veterans Benefits Assistance
+});
+
+const patterns = {
+  '911': '###',
+  default: '###-###-####',
+};
+
+const recordClick = () => {
+  // TO-DO: pass event name & location URL
+  // recordEvent({ event: `telephone-click` });
+};
+
+// Allow labels "gi-bill", "GI_Bill" or "GI BILL"
+const formatLabel = label => label?.replace(/[-_\s]/g, '').toLowerCase();
```

Comment:
I'd actually prefer if we _didn't_ allow for this level of flexibility when passing in known numbers. I mean, if we have the constants we should try to enforce using them? But I'm open to being convinced otherwise!

Reply:
Huh, I thought I removed that.. I'll double check

Reply:
I left it in, but it's meant to clean up the number passed into the `contact` property. You can pass in a number with dashes or spaces, e.g. `1 800 555-1212` and it will still be able to apply it to the pattern.